src/extension.ts

```diff
@@ -255,6 +255,15 @@ function findCachedDataForFile(absolutePath: string): GcovFileData | undefined {
 	if (dataOfFile !== undefined) {
 		return dataOfFile;
 	}
+	/* path of win32 is case insensitive and path separators may be different */
```

Comment:
Style: Use a full sentence as comment, starting with a capital letter and ending with a dot.

Reply:
done